test/e2e/specs/onboarding/vertical__none.ts

```diff
@@ -0,0 +1,140 @@
+/**
+ * @group calyepso-quarantined
+ */
+
+import {
+	DataHelper,
+	DomainSearchComponent,
+	LoginPage,
+	MyHomePage,
+	NewUserResponse,
+	RestAPIClient,
+	SignupPickPlanPage,
+	StartSiteFlow,
+	UserSignupPage,
+} from '@automattic/calypso-e2e';
+import { Page, Browser } from 'playwright';
+import { apiCloseAccount } from '../shared';
+
+declare const browser: Browser;
+
+describe( DataHelper.createSuiteTitle( 'Vertical: None, Something Else, or Skip' ), function () {
+	const testUser = DataHelper.getNewTestUser( {
+		usernamePrefix: 'free',
+	} );
+
+	let domainSearchComponent: DomainSearchComponent;
+	let page: Page;
+	let startSiteFlow: StartSiteFlow;
+	let userCreatedFlag = false;
+	let userDetails: NewUserResponse;
+
+	beforeAll( async () => {
+		page = await browser.newPage();
+	} );
+
+	describe( 'Onboarding flow', function () {
+		it( 'Navigate to Signup page', async function () {
+			const loginPage = new LoginPage( page );
+			await loginPage.visit();
+			await loginPage.clickSignUp();
+		} );
+
+		// TODO: Investigate if using an existing user simplifies the flow
```

Comment:
**praise**: You've raised a logical question here - would using an existing user help eliminate some unnecessary steps from the spec? Indeed, it would - and it's a lot easier than expected to do so. Try this:

```
https://wordpress.com/setup/vertical?siteSlug={your_site_slug_here}
```

This should take you straight to the verticals selection screen.

**question:bonus**: would you consider this to be a valid approach to reduce/eliminate flakiness?